example-config.json

```diff
@@ -30,7 +30,9 @@
         "port":51233
     },
     "log_level":"debug",
-    "log_file":"./clio.log",
+    "log_to_console": true,
+    "log_to_file": true,
+    "log_directory":"./clio_log/",
```

Comment:
Let's make the rotation configurable as well, with the current parameters just used as defaults. So operators can specify how big the files can get before rotating, or how often to rotate, and the max directory size.